drivers/virtualbox/virtualbox.go

```diff
@@ -386,6 +376,10 @@ func (d *Driver) Start() error {
 
 	switch s {
 	case state.Stopped, state.Saved:
+		d.SSHPort, err = setPortForwarding(d.MachineName, 1, "ssh", "tcp", 22, d.SSHPort)
```

Comment:
are you guys detecting the current cfg atm? it may eventually be useful to detect that interface 1 isn't cfg'd the way machine expects and error out (but its going to be rare)